response_operations_ui/controllers/survey_controllers.py

```diff
@@ -30,3 +30,25 @@ def get_survey(short_name):
 
     logger.debug('Successfully retrieved survey', short_name=short_name)
     return response.json()
+
+
+def get_surveys_dict():
+    # TODO get and cache this dictionary at app startup
+    logger.debug('Retrieving surveys list')
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/survey/surveys'
```

Comment:
This code is not DRY , we repeat the code in get_service_list and get_service_dict . We should only be doing this in one place . This will also make it easier when we add caching of this list which I am assuming is a follow up ticket ?